lib/config.go

```diff
@@ -18,9 +22,9 @@ func FileExists(filename string) bool {
   return true
 }
 
-func ReadConfig() (Config, error) {
+func ReadConfig(config_path string) (Config, error) {
   var config Config
-  configPath := getConfigPath()
+  configPath := getConfigPath(config_path)
```

Comment:
In Golang I use camelCase instead of underscores for variables (as is mentioned in Google's style guide).

https://google.github.io/styleguide/go/guide#mixed-caps